clang/include/clang/Basic/Attr.td

```diff
@@ -1474,6 +1482,22 @@ def IntelFPGANumBanks : Attr {
   }];
 }
 
+def IntelFPGAMaxConcurrency : InheritableAttr {
+  let Spellings = [GNU<"max_concurrency">, CXX11<"intelfpga","max_concurrency">];
```

Comment:
Perhaps `intelfpga` would be clearer if written `intel_fpga`.
Are these attributes defined to be compatible with some other tools or are just brand new attributes?

Reply:
> Are these attributes defined to be compatible with some other tools or are just brand new attributes?

You can already use them, for example, in Intel® FPGA SDK for OpenCL™ 19.1.

Reply:
Interesting. With the "intelfpga::" namespace?
But honestly I have no problem about having Intel extension harder to use or unclear. ;-)